comment_scan.rb

```diff
@@ -34,6 +34,13 @@ def matches_bot(bot)
   bot.nil? || bot == '*' || BOT_NAMES.include?(bot.downcase)
 end
 
+def percent_str(numerator, denominator)
+  if denominator == 0
+    return '-'
+  end
+  return "#{(numerator*100/denominator).round(8)}%"
+end
+
```

Comment:
This belongs in `comment_scan/helpers.rb`. Also, some small ruby things -- you can inline that if statement (`return '-' if denominator.zero?`), and the second return isn't needed.

Reply:
Also, you should convert either the numerator or denominator to a float before dividing to avoid automatic rounding to an integer.

Reply:
@thesecretmaster Ahh I should've found `helpers`, that's on me. So you're saying the whole function body can become:

```
return '-' if denominator.zero?
"#{(numerator*100.0/denominator).round(8)}%"
```

Would the `100.0` be enough to make the whole operation floating point like it would in Python?